user/serializers.py

```diff
@@ -1 +1,29 @@
-# write your code here
+from django.contrib.auth import get_user_model
+from rest_framework import serializers
+
+
+class UserSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = get_user_model()
+        fields = ("id", "username", "email", "password", "is_staff",)
+        read_only_fields = ("id", "is_staff",)
+        extra_kwargs = {
+            "password": {
+                "write_only": True,
+                "min_length": 5,
+            },
+        }
+
+    def create(self, validated_data):
+        """create user with encrypted password"""
```

Comment:
The docstring for the `create` method should be capitalized and punctuated correctly. Consider changing it to: 'Create a user with an encrypted password.'